improver/synthetic_data/set_up_test_cubes.py

```diff
@@ -306,7 +306,7 @@ def _construct_dimension_coords(
     realizations: Optional[Union[List[float], ndarray]] = None,
     vertical_levels: Optional[Union[List[float], ndarray]] = None,
     pressure: bool = False,
-    height: bool = True,
+    height: bool = False,
```

Comment:
I think this function contains some residual default height code that might want to be changed to something more generic, e.g. instead of height_coord it could be diagnostic_coord (or whatever you think is most appropriate).
Along with the comments that reference height specifically.

This appears to be the only function that has this issue.